bulk-model-sync-gradle/src/main/kotlin/org/modelix/model/sync/bulk/gradle/tasks/ExportFromModelServer.kt

```diff
@@ -76,13 +85,26 @@ abstract class ExportFromModelServer @Inject constructor(of: ObjectFactory) : De
             val root = branch.getRootNode()
             logger.info("Got root node: {}", root)
             val outputDir = outputDir.get().asFile
-            root.allChildren.forEach {
-                val nameRole = BuiltinLanguages.jetbrains_mps_lang_core.INamedConcept.name
-                val fileName = it.getPropertyValue(nameRole)
+
+            getIncludedModules(root).forEach {
+                val fileName = it.getPropertyValue(BuiltinLanguages.jetbrains_mps_lang_core.INamedConcept.name)
                 val outputFile = outputDir.resolve("$fileName.json")
                 ModelExporter(it).export(outputFile)
             }
         }
+        client.close()
```

Comment:
As long as this is not closeable, shouldn't this happen in a try / finally block to be sure that closing always happens?